testing_monthly_batch_emr.sh

```
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+
+# Hard code values for spark job
+parquet_out="s3a://dpla-provider-export/"
```

Shellcheck (0.11.0)

[warning] parquet_out appears unused. Verify use (or export if used externally).

(SC2034)